doc/Learn/Mvux/toc.yml

```diff
@@ -1,7 +1,7 @@
 
 - name: Overview
   href: xref:Uno.Extensions.Mvux.Overview
-- name: Upgrading Extensions Reactive
+- name: Upgrading MVUX
   href: xref:Uno.Extensions.Reactive.Upgrading
```

Comment:
@ajpinedam I think @kazo0 was talking also about the title for the documentation page itself: https://github.com/unoplatform/uno.extensions/blob/main/doc/Learn/Mvux/reactive-migration.md/#L1

Reply:
@kazo0  could you validate if your comment was about the title on the navigation or the title on the actual page

Reply:
Fixed